Dynatrace/dynatrace-operator
# Remove enabled field for extensions

## Description

Remove the `.enabled` flag from the `.extensions` section. Extensions shall be enabled by using

```
 extensions: {}
```

If the extensions section is missing, no extension-related components shall be deployed.

## How can this be tested?

* Deploy a Dynakube with extensions enabled with the new schema. Check that EEC and collector are deployed
* Deploy a Dynakube without an extensions section. Check that EEC and collector are not deployed
